src/connectors/dispatcher/impl.js

```diff
@@ -8,7 +8,17 @@ const {host, insecure, auth} = require('../../config').dispatcher;
 const Connector = require('../Connector');
 const metrics = require('../metrics');
 
-const QSOPTIONS = { encode: false };
+const FILTEROPTIONS = { encode: true };
+const FIELDOPTIONS = { encode: true, indices: false };
```

Comment:
filter does not use indices either i.e. you can simplify by having one options constant (with indices disabled)